libs/multiplayer/player.ts

```diff
@@ -0,0 +1,672 @@
+//% color="#207a77"
+//% icon="\uf0c0"
+//% block="Multiplayer"
+namespace mp {
+    const MAX_PLAYERS = 4;
+
+    export enum PlayerSlot {
+        //% block="player 1"
+        One = 1,
+        //% block="player 2"
+        Two = 2,
+        //% block="player 3"
+        Three = 3,
+        //% block="player 4"
+        Four = 4
+    }
+
+    export enum PlayerProperty {
+        //% block="index"
+        Index = 1,
+        //% block="slot"
```

Comment:
I'm not sure I'd get what slot means here necessarily on the block -- could this maybe be something `PlayerNumber` / on the block as just `player number`? Maybe we should ask on forum to see if people 'get' it?

Reply:
just _number_ would do.